Core/GDCore/Project/Object.h

```diff
@@ -70,6 +72,13 @@ class GD_CORE_API Object {
     return gd::make_unique<gd::Object>(*this);
   }
 
+  /**
+   * \brief Return the object configuration.
+   */
+  gd::ObjectConfiguration& GetConfiguration();
+
+  gd::ObjectConfiguration& GetConfiguration() const;

```

Comment:
This should return a `const gd::ObjectConfiguration&`:
```suggestion
  const gd::ObjectConfiguration& GetConfiguration() const;

```